MarsQACompetitionTask/POMTests/POM.cs

```diff
@@ -0,0 +1,103 @@
+using MarsQACompetitionTask.Utilities;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+//using OpenQA.Selenium.DevTools.V114.Profiler;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace MarsQACompetitionTask.POMTests
+{
+    [TestFixture]
+    public class POM : CommonDriver
+    {
+        /*public IWebDriver driver;
+        public SignIn SignInObj;
+        public Education EducationObj;
+        public Certifications CertificationObj;*/
+        
+
+        [Test, Order(1), Description("Create Record")]
+        public void CreateEducationTests()
+        {
+            SignInObj.SigninStep();
+            EducationObj.EducationTab();
+            EducationObj.CreateEducation();      
+        }
+
+        [Test, Order(2), Description("Cancel Record")]
+        public void CancelEducationTests()
+        {
+            SignInObj.SigninStep();
+            EducationObj.EducationTab();
+            EducationObj.CancelAddedEducation();
+        }
+        [Test, Order(3), Description("Edit or update Record")]
+        public void UpdateEducationTests()
+        {
+            SignInObj.SigninStep();
+            EducationObj.EducationTab();
+            EducationObj.EditEducation();
+            
+        }
+        [Test, Order(4), Description("CancelUpdated Record")]
+        public void CancelUpdatedEducationTests()
+        {
+            SignInObj.SigninStep();
+            EducationObj.EducationTab();
+            EducationObj.CancelEditEducation();
+        }
+        [Test, Order(5), Description("Delete Record")]
+        public void DeleteEducationTests()
+        {
+            SignInObj.SigninStep();
+            EducationObj.EducationTab();
+            EducationObj.DeleteEducation();
+            Thread.Sleep(1000);
+        }
+        [Test, Order(6), Description("Create CRecord")]
+        public void CreateCertificationsTests()
+        {
+            SignInObj.SigninStep();
+            CertificationObj.CertificationTab();
+            CertificationObj.CreateCertification();
+        }
+
+        [Test, Order(7), Description("Cancel Record")]
+        public void CancelAddedCertifications()
+        {
+            SignInObj.SigninStep();
+            CertificationObj.CertificationTab();
+            CertificationObj.CancelAddedCertification();
+        }
+        [Test, Order(8), Description("Edit or update Record")]
+        public void UpdateCertification()
+        {
+            SignInObj.SigninStep();
+            CertificationObj.CertificationTab();
+            CertificationObj.EditCertification();
+
+        }
+        [Test, Order(9), Description("CancelUpdated Record")]
+        public void CancelUpdatedCertificationTests()
+        {
+            SignInObj.SigninStep();
+            CertificationObj.CertificationTab();
+            CertificationObj.CancelEditCertification();
```

Comment:
Where are your asserts can you please add asserts here, if they are in pages they need to be moved to this test level

Reply:
is this done?